components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -144,6 +144,12 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
         }
     }
 
+    public handleKeyDown = (event: KeyboardEvent): void => {
+        if (event.key === 'Enter' && this.state.show) {
```

Comment:
There is a value in `Constants.KeyCodes.ENTER` that you can use instead of using the string `'Enter'`.